python/ray/serve/backend_state.py

```diff
@@ -91,11 +94,20 @@ def __set_state__(self, d: Dict[Any, Any]) -> None:
         self._ready_obj_ref = None
         self._drain_obj_ref = None
 
+    @property
+    def replica_tag(self) -> str:
+        return str(self._replica_tag)
```

Comment:
why do we need the `str()`?

Reply:
ah i used to have it as the class instance, it's not needed now.